### ISSUE TITLE
Optional chaining Error in @javelin/ecs module output

### ISSUE THREAD

**Issue by kenjinp** [3mcd/javelin#171]
from version `1.0.0-alpha.8` and up It looks like your transpilation step doesn't convert optional chains. This makes the package unusable for node versions 12 and lower. I think it could be an easy task to change the output target to support node 12. 

```@javelin/ecs/dist/cjs/effects/core/use_monitor.js:105
        if (_query !== query && !_query?.equals(query)) {
                                        ^

SyntaxError: Unexpected token '.'```

**Comment by 3mcd** [comment]
Good catch. I'll try and get this in ASAP. I'm moving a little slow due to some life changes but I should have more time after a week or two!

**Comment by kenjinp** [comment]
it seems they are fixed as of `1.0.0-alpha.12`, thanks!
